Lib/email/mime/image.py

```diff
@@ -38,6 +41,12 @@ def __init__(self, _imagedata, _subtype=None,
         header.
         """
         if _subtype is None:
+            fullname = f"{__name__}.{self.__class__.__qualname__}"
+            msg = ("The *_subtype* argument's default value for {name}"
+                   "is deprecated due to the deprecation of imghdr (slated for "
+                   "removal in Python {remove}); explicitly specify the image "
+                   "subtype")
```

Comment:
This is telling people who _aren't_ using `_subtype`, to start using this "private" argument.

Should it be promoted to a "public" argument? Or not worth the effort?

Reply:
It's not private based on the documentation explicitly referencing it. I think this is part of how the `email` package exposes the class via some other mechanisms.